README.md

```diff
@@ -960,6 +960,61 @@ cable.subscribe("aCardWasUpdated { slug }", {
 
 A working JavaScript code sample is available in [examples/subscribeAllCardUpdates.js](./examples/subscribeAllCardUpdates.js).
 
+#### Current user updates
+
+The below example will start a subscription for updates on the current user and exit on the first of the following events:
+
+1. Connection failures or error
+2. An update is received for the current user
+
+```js
+const { ActionCable } = require("@sorare/actioncable");
+
+const cable = new ActionCable({
+  headers: {
+    // 'Authorization': `Bearer <YourJWTorOAuthToken>`,
+    // 'APIKEY': '<YourOptionalAPIKey>'
+  },
+});
+
+cable.subscribe('currentUserWasUpdated { slug nickname }', {
+  connected() {
+    console.log("connected");
+  },
+
+  disconnected(error) {
+    console.log("disconnected", error);
+    process.exit(1);
+  },
+
+  rejected(error) {
+    console.log("rejected", error);
+    process.exit(1);
+  },
+
+  received(data) {
+    if (data?.result?.errors?.length > 0) {
+      console.log('error', data?.result?.errors);
+      process.exit(1);
+      return;
+    }
+    const currentUserWasUpdated = data?.result?.data?.currentUserWasUpdated;
+    if (!currentUserWasUpdated) {
+      return;
+    }
+    const { slug } = currentUserWasUpdated;
+    console.log('current user was updated', slug);
+    process.exit(0);
+  }
+});
+
+```
+
+This example can be found in [examples/subcribeCurrentUserUpdates.js](./examples/subscribeCurrentUserUpdates.js) which can be run with the following environment parameters:
```

Comment:
I feel like linking the script here is enough and we shouldn't add code to the README as it's already big, wdyt?

Reply:
Mmmmmm the idea of this read me is that people could get enticed to copy paste it and make it clean. If we only leave the reference file they could be tempted to keep the environment variable logic (which may not be the best idea).

But I give you that it's debatable IMHO. WDYT ?